build.sbt

```
@@ -137,11 +135,33 @@ lazy val commonSettings = Seq(
   // uncomment for full stack traces
   // Test / testOptions   += Tests.Argument("-oDF"),
   Test / fork          := true,
-  resolvers            ++= Resolver.sonatypeOssRepos("public"),
+  resolvers            += Resolver.sonatypeCentralSnapshots,
+  resolvers            += Resolver.sonatypeCentralRepo("releases"),
   resolvers            += Resolver.mavenLocal,
   resolvers            += "broad-snapshots" at "https://broadinstitute.jfrog.io/artifactory/libs-snapshot/",
   shellPrompt          := { state => "%s| %s> ".format(GitCommand.prompt.apply(state), version.value) },
   updateOptions        := updateOptions.value.withCachedResolution(true),
+  pomExtra             := <url>https://github.com/fulcrumgenomics/fgbio</url>
+    <licenses>
+      <license>
+        <name>MIT License</name>
+        <url>http://www.opensource.org/licenses/mit-license.html</url>
+      </license>
+    </licenses>
+    <developers>
+      <developer>
+        <id>nh13</id>
+        <name>Nils Homer</name>
+        <url>http://github.com/nh13</url>
+        <email>nils@fulcrumgenomics.com</email>
+      </developer>
+      <developer>
+        <id>tfenne</id>
+        <name>Tim Fennell</name>
+        <url>http://github.com/tfenne</url>
+        <email>tim@fulcrumgenomics.com</email>
+      </developer>
```

I would welcome that!

Suggestion includes `s/http/https/` in the URLs too:

```suggestion
    <licenses>
      <license>
        <name>MIT License</name>
        <url>https://www.opensource.org/licenses/mit-license.html</url>
      </license>
    </licenses>
    <developers>
      <developer>
        <id>nh13</id>
        <name>Nils Homer</name>
        <url>https://github.com/nh13</url>
        <email>nils@fulcrumgenomics.com</email>
      </developer>
      <developer>
        <id>tfenne</id>
        <name>Tim Fennell</name>
        <url>https://github.com/tfenne</url>
        <email>tim@fulcrumgenomics.com</email>
      </developer>
      <developer>
        <id>clintval</id>
        <name>Clint Valentine</name>
        <url>https://github.com/clintval</url>
        <email>clint@fulcrumgenomics.com</email>
      </developer>      
```

---

I assigned @tfenne to see if he approves/rejects.

No hard feelings either way.